packages/babel-helpers/src/helpers.js

```diff
@@ -875,6 +875,12 @@ helpers.readOnlyError = helper("7.0.0-beta.0")`
   }
 `;
 
+helpers.writeOnlyError = helper("7.12.12")`
```

Comment:
I just noticed that this should be 7.12.13, because in 7.12.12 the helper isn't available. Do you mind opening a PR? 🙏